examples/map.birb

```diff
@@ -2,4 +2,11 @@ Map map = {
     "content": "This is a message"
 };
 
-scremLn(map["content"]); // output This is a message
\ No newline at end of file
+scremLn(map["content"]); // output This is a message
+scremLn(map.isEmpty); // output false
+scremLn(map.isNotEmpty); // output true
+scremLn(map.length); // output 1
+scremLn(map.hashCode); // ignore
+
+// Methods
+scremLn(map.toString()); // ignore
```

Comment:
Why are you ignoring the output on this line? it should be `{content: This is a message}`

Reply:
Sure